<ui version="4.0" >
 <class>ClientHierarchyViewOptions</class>
 <widget class="QWidget" name="ClientHierarchyViewOptions" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QStackedWidget" name="stackedWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="visibility" >
      <layout class="QGridLayout" >
       <item row="0" column="0" colspan="2" >
        <widget class="QCheckBox" name="showTree" >
         <property name="text" >
          <string>Show Tree</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2" >
        <widget class="QCheckBox" name="showHierarchicalGraph" >
         <property name="text" >
          <string>Show Hierarchical Graph</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0" colspan="2" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>161</width>
           <height>141</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="0" >
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Hierarchical Graph Layout</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1" >
        <widget class="QComboBox" name="hgraphLayout" >
         <item>
          <property name="text" >
           <string>Tree</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>TreeOrbit</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="6" column="0" >
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>Tree Leaf Spacing</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1" >
        <widget class="QDoubleSpinBox" name="leafSpacing" >
         <property name="decimals" >
          <number>1</number>
         </property>
         <property name="maximum" >
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep" >
          <double>0.100000000000000</double>
         </property>
         <property name="value" >
          <double>0.500000000000000</double>
         </property>
        </widget>
       </item>
       <item row="7" column="0" >
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>Tree Radial Angle</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1" >
        <widget class="QSpinBox" name="radialAngle" >
         <property name="maximum" >
          <number>360</number>
         </property>
         <property name="value" >
          <number>120</number>
         </property>
        </widget>
       </item>
       <item row="8" column="0" >
        <widget class="QLabel" name="label_5" >
         <property name="text" >
          <string>Edge Bundling Strength</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1" >
        <widget class="QDoubleSpinBox" name="bundlingStrength" >
         <property name="maximum" >
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep" >
          <double>0.100000000000000</double>
         </property>
         <property name="value" >
          <double>0.250000000000000</double>
         </property>
        </widget>
       </item>
       <item row="9" column="0" >
        <widget class="QLabel" name="label_6" >
         <property name="text" >
          <string>Tree Level Spacing</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1" >
        <widget class="QDoubleSpinBox" name="levelSpacing" >
         <property name="decimals" >
          <number>1</number>
         </property>
         <property name="singleStep" >
          <double>0.100000000000000</double>
         </property>
         <property name="value" >
          <double>1.200000000000000</double>
         </property>
        </widget>
       </item>
       <item row="10" column="0" >
        <widget class="QCheckBox" name="radialLayout" >
         <property name="text" >
          <string>Radial Tree Layout</string>
         </property>
         <property name="checked" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="foo" />
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
